x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/expression/Order.java

```diff
@@ -23,7 +23,11 @@
     }
 
     public enum NullsPosition {
-        FIRST, LAST;
+        FIRST, LAST,
+        /**
+         * Nulls position has not been specified by the user and an appropriate default will be used.
```

Comment:
What is the new/appropriate vs the previous default?

Reply:
I've extended the comment and point to https://github.com/elastic/elasticsearch/issues/77068 now. The default is not particularly meaningful but it stays compatible with the legacy behavior.

Reply:
Thanks. Let's put this on the discussion list for 8.x and whether it is worth reviewing the default so it's consistent across different contexts with minimal (if any) performance penalty.